modules/client-configuration/pages/contact-methods-pushssh.adoc

```diff
@@ -0,0 +1,103 @@
+[[contact-methods-pushssh]]
+= Push via SSH
+
+
+Push via SSH is used in environments where clients cannot reach the {productname} Server directly.
+In this environment, clients are located in a firewall-protected zone called a DMZ.
+No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
+
+The Push via SSH method creates an encrypted tunnel from the {productname} Server on the internal network to the clients located on the DMZ.
+After all actions and events are executed, the tunnel is closed.
+
+The server uses SSH to contact the clients at regular intervals, checking in and performing scheduled actions and events.
+
+
+
+[IMPORTANT]
+====
+Re-installing systems using the provisioning model is not currently supported on clients managed with Push via SSH.
+====
+
+For tunneling connections via SSH, two available port numbers are required, one for tunneling HTTP and the second for tunneling via HTTPS (HTTP is only necessary during the registration process).
```

Comment:
I think we are not using HTTP and `1232`  now at all. Even for the bootstrapping HTTPS with `1233`  is used.
There is a comment from @mcalmer in the changelog of `spacewalk-web`:
```
- remove ssh_push_port_http default configuration as it is used only by
  traditional clients which are now unsupported
```

Reply:
ok, changed.